test/emailVerification/sendVerificationMail.test.js

```diff
@@ -5,7 +5,12 @@ const sinonChai = require('sinon-chai');
 const sgMail = require('@sendgrid/mail');
 const emailTemplate = require('../../lib/utils/emailService/emailTemplate');
 const sendVerificationMail = require('../../lib/utils/emailService/emailVerification');
-const { FROM, SUBJECT, SERVER_ERROR_CODE } = require('../../constants');
+const {
+  FROM,
+  SUBJECT,
+  SERVER_ERROR_CODE,
+  RESOURCE_CREATED_CODE
+} = require('../../constants');
 
 chai.use(sinonChai);
 const validToken = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJ1dWlkIjoiNDE1ZTJlZmEtMTllNy0xMWU5LWFiMTQtZDY2M2JkODczZDkzIiwiaWF0IjoxNTQ3ODIxMjk2LCJleHAiOjE1NDc5MDc2OTZ9.DVwT7E9aU92ByHEw7GNn8URI-iZAR2VCLIY47dilOec';
```

Comment:
Kindly move this validToken to constant file.

Reply:
fixed